src/AudacityApp.cpp

```diff
@@ -987,8 +1020,10 @@ bool AudacityApp::OnExceptionInMainLoop()
 
 AudacityApp::AudacityApp()
 {
+#if defined(USE_BREAKPAD)
+    InitBreakpad();
```

Comment:
Is this an early enough place to initialize?  Is this the earliest possible place?

Maybe the answer is in fact no, but the question should be asked and your answer should be a code comment.

This source file contains the actual definition of main() if you look elsewhere.

Reply:
>This source file contains the actual definition of main() if you look elsewhere.
Isn't it generated by IMPLEMENT_APP for Windows target? I thought that this is a "reasonably early" place to initialize breakpad